_rules/auto-play-audio-does-not-exceed-3-seconds-aaa1bf.md

```diff
@@ -1,9 +1,9 @@
 ---
 id: aaa1bf
-name: auto-play audio does not exceed 3 seconds
+name: '`autoplay` audio does not exceed 3 seconds'
```

Comment:
If autoplay is used as an adjective (?) or a verb, there is no reason for the backticks. In these cases, autoplay does not refer to the autoplay attribute.

To avoid confusion, consider rewriting where autoplay does not refer to the attribute.

Suggestion in this case: Autoplay audio does not exceed 3 seconds -> Audio that plays automatically does not exceed 3 seconds.